src/chainstate/stacks/db/mod.rs

```diff
@@ -665,7 +665,7 @@ const CHAINSTATE_INITIAL_SCHEMA: &'static [&'static str] = &[
     CREATE TABLE staging_blocks(anchored_block_hash TEXT NOT NULL,
                                 parent_anchored_block_hash TEXT NOT NULL,
                                 consensus_hash TEXT NOT NULL,
-                                -- parent_consensus_hash is the consensus hash of the parent sortition of the sortition that chose this block
+                                -- parent_consensus_hash is the consensus hash of the parent sortition that chose this block
```

Comment:
Is this comment update correct? What is the "parent sortition that chose this block"?

Reply:
It's the consensus hash of the sortition of the parent Stacks block.

Reply:
Fixed